src/editor/Renderer.cpp

```diff
@@ -510,15 +510,25 @@ void Renderer::saveSettings() {
 
 void Renderer::loadSettings() {
 
+	if (g_settings.gamedir.size() == 0)
+	{
+		g_settings.gamedir = "./";
+	}
+
 	if (g_settings.fgdPaths.size() == 0) {
-		g_settings.fgdPaths.push_back(g_settings.gamedir + "/svencoop/sven-coop.fgd");
+		g_settings.fgdPaths.push_back(g_settings.gamedir + "svencoop/sven-coop.fgd");
 	}
 
 	if (g_settings.resPaths.size() == 0) {
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop/");
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop_addon/");
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop_downloads/");
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop_hd/");
+		g_settings.resPaths.push_back(g_settings.gamedir + "svencoop/");
+		g_settings.resPaths.push_back(g_settings.gamedir + "svencoop_addon/");
+		g_settings.resPaths.push_back(g_settings.gamedir + "svencoop_downloads/");
+		g_settings.resPaths.push_back(g_settings.gamedir + "svencoop_hd/");
+	}
+
+	if (!g_settings.valid)
+	{
+		return;
 	}

```

Comment:
Are some settings are broken without this? I think defaults should be added here if there's a problem:
https://github.com/wootguy/bspguy/blob/master/src/editor/Renderer.h#L65-L71

I can do that later, too.

Reply:
> Are some settings are broken without this? I think defaults should be added here if there's a problem:
> https://github.com/wootguy/bspguy/blob/master/src/editor/Renderer.h#L65-L71
> 
> I can do that later, too.

Just lost this function, it from You code : https://github.com/wootguy/bspguy/blob/34b85a41e80047e777015e74e53d6477e8197277/src/editor/Renderer.cpp#L501

Reply:
Or need change ?

Reply:
nah it's good, I forgot it was there before.